tests/middleware/test_cors.py

```diff
@@ -245,12 +245,30 @@ def homepage(request):
     assert response.headers["vary"] == "Origin"
 
 
-def test_cors_vary_header_is_properly_set():
+def test_cors_vary_header_is_properly_set_for_credentialed_request():
```

Comment:
should we add another `test_cors_vary_header_is_properly_set_for_not_credentialed_request` ?